test/unit/app/models/edition/publishing_test.rb

```diff
@@ -125,34 +131,14 @@ class Edition::PublishingTest < ActiveSupport::TestCase
     assert edition.errors[:base].include?("This document has not been force-published")
   end
 
-  test "#unpublished? returns false if publicly visible" do
-    published_edition = build(:published_edition)
-
-    assert_not published_edition.unpublished?
-  end
-
-  test "#unpublished? returns false if no unpublishing exists" do
-    draft_edition = build(:draft_edition)
-
-    assert_not draft_edition.unpublished?
-  end
-
-  test "#unpublished? returns true if not publicly visible and unpublishing exists" do
-    unpublishing = build(:unpublishing)
-    draft_edition_with_unpublishing = build(:draft_edition, unpublishing:)
-
-    assert draft_edition_with_unpublishing.unpublished?
-  end
-
   test "#unpublished_edition returns nil if not unpublished" do
     edition = build(:published_edition)
 
     assert_nil edition.unpublished_edition
   end
 
-  test "#unpublished_edition returns unpublished edition if unpublished" do
-    edition = create(:draft_edition, :with_document)
-    edition.build_unpublishing(edition:)
+  test "#unpublished_edition returns itself if unpublished" do
+    edition = create(:unpublished_edition)
```

Comment:
This is weird and this whole method needs to be refactored away, but I will add a separate PR for that